integration-test.sh

```diff
@@ -15,7 +15,7 @@ export MINIMIZE_DOWNTIME=0
 
 if [[ "$test_option" == "--initial-install" ]]; then
   echo "Testing initial install"
-  source _integration-test/run.sh
+  pytest --reruns 5 _integration-test/run.py
```

Comment:
Is that even needed? If we have proper retries when polling data, we likely don't need to retry the tests up to 5 times, do we?